news/2024/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "The Followpoint: dressurf, Top Keyboard Smasher"
```

Comment:
bit different from the last one

![image](https://github.com/ppy/osu-wiki/assets/36758269/0980f65a-2171-4297-a510-47a063354e21)

```suggestion
title: "The Followpoint: dressurf, The #1 Key Smasher"
```